editor/index.html

```diff
@@ -106,6 +106,12 @@ <h1>p5.fab</h1>
         <input id="file-upload" type="file" />
       </div>
       <div class="right">
+        <div class="menu-item">printer
+          <div class="menu-item-dropdown-content">
+            <i onclick="setPrinter('ender3-pro')">Ender 3 Pro</i>
+            <i onclick="setPrinter('q5')">Flsun Q5</i>
+          </div>
+        </div> 
         <div class="menu-item" onclick="updatePreview()">visualize</div>
```

Comment:
I'd suggest changing this to "refresh". "visualise" confused me as if you already printed something once you have to "visualise" to print again. I didn't want to step out of line and change your UI though